api/buildstatus/buildstatus_controller.go

```diff
@@ -0,0 +1,72 @@
+package buildstatus
+
+import (
+	"net/http"
+
+	"github.com/equinor/radix-api/api/utils"
+	"github.com/equinor/radix-api/models"
+	"github.com/gorilla/mux"
+)
+
+const rootPath = "/applications/{appName}"
+
+type buildStatusController struct {
+	*models.DefaultController
+}
+
+// NewBuildStatusController Constructor
+func NewBuildStatusController() models.Controller {
+	return &buildStatusController{}
+}
+
+// GetRoutes List the supported routes of this handler
+func (dc *buildStatusController) GetRoutes() models.Routes {
+	routes := models.Routes{
+		models.Route{
+			Path:        rootPath + "/buildstatus/{env}",
```

Comment:
The route for other environment specific requests is /applications/{appName}/environments/{envName}/[deployments/events/components].
It would be more consistent to use the same route pattern for buildstatus